src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/AdherenceUtils.java

```diff
@@ -73,7 +75,7 @@ public static int calculateAdherencePercentage(Collection<EventStream> streams)
     
     public static ParticipantStudyProgress calculateProgress(AdherenceState state, List<EventStream> eventStreams) {
         if (state.getMetadata().isEmpty()) {
-            return ParticipantStudyProgress.NO_SCHEDULE;
+            throw new EntityNotFoundException(Schedule2.class);
         }
```

Comment:
Removed this, as I don't believe it is necessary. It'll just calculate that an empty schedule is unstarted, and elsewhere in the code you don't get here if there's no schedule, because you never generate the report. Adjusted test to show this is what happens (UNSTARTED...an empty schedule can't really be started or completed).